pass-deposit-services/deposit-core/src/main/resources/application.properties

```diff
@@ -20,11 +20,13 @@ pass.client.url=${PASS_CLIENT_URL:localhost:8080}
 pass.client.user=${PASS_CLIENT_USER:fakeuser}
 pass.client.password=${PASS_CLIENT_PASSWORD:fakepassword}
 
-ftp.host=${FTP_HOST:localhost}
-ftp.port=${FTP_PORT:21}
+ftp.host=${PMC_FTP_HOST:localhost}
```

Comment:
These properties should be `pmc.` as a prefix

Reply:
Done